### ISSUE TITLE
Database errors

### ISSUE THREAD

**Issue by ShaeBrown** [teamrandint/central#26]
Transaction server is reporting some errors for the database - needs some investigating:

Error connecting to the database to push buy command: strconv.ParseInt: parsing "282438.29000000000002046": invalid syntax

Could not add stock to reserve: redigo: unexpected type for Float64, got type int64
Error getting reserved stock from database: strconv.ParseFloat: parsing "ZSY:155.34:1": invalid syntax

Error getting reserved stock from database: redigo: nil returned

**Comment by jarredhawkins** [comment]
Looks like a type issue with the reserve stocks and a floating point calc done somewhere with the parsing error.

**Comment by ShaeBrown** [comment]
Yeah..  :( also alot of nil returns. Not sure if they should be there (like when the person hasn't bought any stock yet). Maybe 0 should be returned?
